packages/pluggableWidgets/range-slider-web/src/utils/isVertical.ts

```diff
@@ -0,0 +1,5 @@
+import { RangeSliderContainerProps } from "../../typings/RangeSliderProps";
+
+export function isVertical(params: Pick<RangeSliderContainerProps, "orientation">): boolean {
+    return params.orientation === "vertical";
+}
```

Comment:
Nit: do we really need this is as a separate util instead of just inline ` === "vertcal"`?

Reply:
We don't, but this is left over and I think this is fine leave this as it is for now